### DISCUSSION TITLE
Want RSS feed for /advisories (Security Advisories)

### DISCUSSION THREAD

**Discussion by usagiga** [community/community#7517] [Code Security]
[/advisories](https://github.com/advisories) is pretty good, but it has no way to subscribe such as RSS feed. 😢
I want to read it from RSS reader.

**Comment by usagiga** [comment]
Finally I found official webhook!
We can use it instead of RSS feed.
https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#security_advisory

**Reply by vorotech** [reply]
The problem with the webhook that it can be utilized only by repo owners.
But it would be nice to have the RSS/ATOM feed available for consumers of the repo.
E.g. https://github.com/discourse/discourse/security/advisories can be useful for admins who maintain the Discourse blog.

**Comment by mkurz** [comment]
I have a similiar question/request:
When a user watches _only_ "Security alerts" for a repository, shouldn't that user get a notification from GitHub as soon a Security Advisory is published for that repo? Right now no notification will be send at all.
<kbd><img width="400" src="https://user-images.githubusercontent.com/644927/161352373-ed984b0b-f6c8-4f43-b4d0-be1a1d18f3ec.png" /></kbd>

**Reply by mkurz** [reply]
Refering the relevant discussion: https://github.com/orgs/community/discussions/12226#discussioncomment-10142233

**Comment by StayPirate** [comment]
having something like: https://github.com/nextcloud/security-advisories/security/advisories.atom or https://github.com/nextcloud/security-advisories/security/advisories.rss would be very useful for many people/use-cases. I'm part of the security-team of a widely used downstream Linux distribution. It would be very good for us to be able to subscribe to the Security Advisory feed of projects we repackage. @github-staff can you consider to implement this feature?

**Comment by tuggan** [comment]
Just replying to keep this alive. I think @StayPirate has the right approach from my perspective.

**Comment by andresanches** [comment]
Bumping to keep the issue alive. For my use case it would be helpful to watch RSS or Atom feeds in my company's slack, which reaches a wide engineering audience.

**Comment by jduck** [comment]
Did you try https://github.com/security-advisories ?

**Reply by DanielRuf** [reply]
Is there some documentation to filter the results? I'm looking for something like https://github.com/advisories?query=node.js but as RSS / Atom feed.

**Comment by DanielRuf** [comment]
Found https://azu.github.io/github-advisory-database-rss/ which is more like what I am looking for.

**Comment by alon710** [comment]
I've been following this discussion and noticed many of you are looking for a more accessible way to track GHSAs and vulnerabilities via RSS.

I built [cvereports.com](https://cvereports.com) to solve exactly this. It automatically generates a blog post for every new advisory and provides a dedicated RSS feed that you can plug into Slack, Discord, or your favorite reader.

Beyond just the feed, it uses a custom AI agent integrated with multiple security tools to generate detailed reports for new CVEs and GHSAs. Hope this helps bridge the gap until GitHub adds native RSS support!
